### PR TITLE
SEO-191923--wpf-page-with-redirect

### PR DESCRIPTION
**Pull request by hillary-ochieng**
Hi @Yvone-Atieno 

|Title|Description|
|--|--|
|Task Category|Site Audit - Page with redirect|
|Content Task Link/Mail Screenshot| NA|
|UX task| NA|
|Hotfix PR|https://github.com/syncfusion-content/wpf-docs/pull/1555|
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/191923|
|Excel/SharePoint link| NA|

|Changes made|Reason for changes|
|--|--|
|Changed redirect link  |  To avoid redirection 301 or 302,providing direct link    |     

Regards,
Hillary

### CODE REVIEW

File: wpf/TreeGrid/Export-To-Excel.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The following assemblies should be added for exporting a tree grid to Excel:-
...
 
-You can export the tree grid to Excel by using the [ExportToExcel](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.Converter.TreeGridExcelExportExtension.html#Syncfusion_UI_Xaml_TreeGrid_Converter_TreeGridExcelExportExtension_ExportToExcel_Syncfusion_UI_Xaml_TreeGrid_SfTreeGrid_Syncfusion_UI_Xaml_TreeGrid_Converter_TreeGridExcelExportingOptions_) extension method present in [Syncfusion.UI.Xaml.TreeGrid.Converter](https://help.syncfusion.com/cr/wpf/Syncfusion.html) namespace.
+You can export the tree grid to Excel by using the [ExportToExcel](html#Syncfusion_UI_Xaml_TreeGrid_Converter_TreeGridExcelhttps://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.Converter.TreeGridExcelExportExtension.ExportExtension_ExportToExcel_Syncfusion_UI_Xaml_TreeGrid_SfTreeGrid_Syncfusion_UI_Xaml_TreeGrid_Converter_TreeGridExcelExportingOptions_) extension method present in [Syncfusion.UI.Xaml.TreeGrid.Converter](https://help.syncfusion.com/cr/wpf/Syncfusion.html) namespace.
```
Comment (Yvone-Atieno) [inline wpf/TreeGrid/Export-To-Excel.md:22]: Check the updated link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: wpf/TreeGrid/Export-To-Excel.md (hunk 1 above)
Comment (hillary-ochieng) [reply wpf/TreeGrid/Export-To-Excel.md]: Updated proper link

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**